src/triage/component/catwalk/model_testers.py

```diff
@@ -73,9 +78,34 @@ def process_model_test_task(self, test_store, train_store, model_ids):
             max(as_of_times),
             len(as_of_times),
         )
+
+
         for model_id in model_ids:
             logging.info("Testing model id %s", model_id)
 
+            test_eval_start_time = test_store.as_of_dates[0]
+            test_eval_end_time = test_store.as_of_dates[-1]
+            test_as_of_date_frequeny = test_store.metadata["as_of_date_frequency"]
+            test_eval_obj = test_store.matrix_type.evaluation_obj
+
+            session = self.sessionmaker()
```

Comment:
I'm not sure there's any advantage to creating a new session for every loop / `model_id`, is there?